mmv1/products/gkehub2/Feature.yaml

```diff
@@ -170,6 +180,84 @@ properties:
                       - :MODE_UNSPECIFIED
                       - :COPY
                       - :MOVE
+  - !ruby/object:Api::Type::NestedObject
+    name: fleetDefaultMemberConfig
+    description: Optional. Fleet Default Membership Configuration.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: mesh
+        description: Service Mesh spec
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: management
+            description: 'Whether to automatically manage Service Mesh'
+            values:
+              - :MANAGEMENT_UNSPECIFIED
+              - :MANAGEMENT_AUTOMATIC
+              - :MANAGEMENT_MANUAL
+            required: true
+      - !ruby/object:Api::Type::NestedObject
+        name: configmanagement
```

Comment:
Can you specify more of these fields in the tests? We try to have as many of the fields set and then changed as possible.